### PR TITLE
Added copy paste code and fixed broken untested code

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Basically I added the copy-paste functionality, then noticed a bunch of messed up code, like `invenroty_set` instead of `inventory_set`. Also fixed the `_set_or_give_wand` func to give proper errors when setting wand from command.

Also added translations for some stuff that didnt have translations yet (mostly from #15 )

Also added extra argument to `_print()` to specify whether or not this is an error.

### CODE REVIEW

**Comment by BisUmTo** [comment]
I would prefer a `_error_print()` or a `_error()` function instead of a flag... 🤔

File: world-edit.sc [hunk 1]
```diff
@@ -608,10 +608,11 @@ _translate(key, ... replace_list) -> (
...
 );
 
-_print(player, fatal, key, ... replace) -> (
+_print(player, key, ... replace) ->
     print(player, format(_translate(key, replace)));
-    if(fatal,exit())
-);
+
+_error(player, key, ... replace)->
+    exit(print(player, format(_translate(key, replace))));
```
Comment (Firigion) [inline world-edit.sc]: Yeah, I like it.

**Review by Firigion — APPROVED** [review]
I'm still not sure how to do and finish a rewview without mentioning a specific aprt of the code :P

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ existing code, so you know what to do right off the bat, but here is the tl;dr,
     - Your function probably needs to access the player's positions. This function will throw the appropriate errors if
```
Comment (replaceitem) [inline CONTRIBUTING.md:32]: Would write 'Your function probably needs to access the player's selection' here too

**Review by replaceitem — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Ghoulboy78) [reply CONTRIBUTING.md:32]: true dat

Comment (Ghoulboy78) [reply CONTRIBUTING.md:32]: but tbh ill do in next pr cos i messed up the merge rule

**Pull request merged**